### PR TITLE
Added lecture files + added name of round-table

### PR DESCRIPTION
**Pull request by bamorim-bio**
#13

### CODE REVIEW

File: docs/2024/README.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ For any inquiries contact [HAAM Community](mailto:haamcommunity2023@gmail.com) o
 - **Lecture** Introduction to Population Genetics: _Stephan Schiffels_
 - **Practical** Data Curation: Poseidon: _Clemens Schmid_
 - **Practical** Unsupervised Analyses: PCA/MDS, Admixture: _Jazmín Ramos-Madrigal_
-- **Round table** _Jazmín Ramos-Madrigal_
+- **Round table** Unsupervised clustering analyses & Ancient DNA: _Jazmín Ramos-Madrigal_
```
Comment (TCLamnidis) [inline docs/2024/README.md:184]: This should be moved to the practical sessions. Jazmín joined for the roundtable as well, but she ran a proper practical session, which is where the slides are from.

**Review by TCLamnidis — CHANGES_REQUESTED** [review]
The rendering looks good! 
I think it would be easier to navigate this if it were in sequence, Day 1 -> Day 2 -> Day3, instead of separating theoretical, practical and round table sessions. We could denote the type of session in the title, or with an emoji or sth. :)

**Comment by bamorim-bio** [comment]
Check if there's anything else you'd like me to change? :)

**Review by TCLamnidis — APPROVED** [review]

**Comment by TCLamnidis** [comment]
Made some tweaks:
 - Marked Robert's session as Theoretical. Was originally planned as a practical session, but due to technical issues it ended up being a lecture instead. 
 - Changed the Day 3 round-table title. and noted that Robert also joined.
 - Added a short explanation to title colours in teaching material page.
 - minor tweaks to HTML styling.

**Pull request merged**